src/Faker/Provider/it_IT/Person.php

```diff
@@ -21,70 +21,73 @@ class Person extends \Faker\Provider\Person
         '{{firstNameFemale}} {{lastName}}',
         '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
     );
-
+    
+    /**
+     * Most common italian male firstname
+     * @link http://www.nomix.it/nomi-italiani-maschili-e-femminili.php
```

Comment:
This site copyrights its data. Publishing it in Faker would be theft. Please find a datasource compatible with our MIT license.

Reply:
Sorry man, but before my PR there wasn't no names datasource in this file, so why should I include it? I'm italian, I can assure you that old names (like Aaron, Akira, Dimitri, Xavier, Odino, Serse, Jelena, Kayla, Thea, Yvonne, Sue ellen and others) aren't used in Italy and you can trust me if I say you that names I've added are most common. Honestly, I wouldn't know where I could find a MIT datasource with these names, so if you don't want to merge this PR I will use my own fork. I mean I've spent time to improve the `it_IT` provider you wrote, if you don't appreciate my job I can't keep working on it.
So please, tell me where I should find the datasource and I will happy to mention it. 

Thanks.

Reply:
I don't see the PorcoDio name.

Reply:
Maybe we can use this list? https://gist.github.com/pdesterlich/2562329

Reply:
I use my own repo datasource. 😎

Reply:
Doesn't work. You may have licensed your repo under MIT, but it looks like you've taken the information somewhere else. This doesn't allow you to publish it under the MIT license.

Reply:
This is false, becausa I made my own list with different names. By the way, could you give an answer to this question? https://github.com/fzaninotto/Faker/pull/1279/files/709e86add3aa828655294ec8ef1537bfde8262a2#r136514555